tests/lib/bin/retry-tool

```diff
@@ -40,33 +40,42 @@ attempt. On failure the exit code from the final attempt is returned.
         default=1,
         help="grace period between attempts (default %(default)ss)",
     )
+    parser.add_argument(
+        "--quiet",
+        dest="verbose",
```

Comment:
IMHO it'd be better to have quiet flag for consistency and reverse the conditionals below. This will also make it cleaner if we ever want to introduce --verbose for being really verbose.

Reply:
Perhaps I misunderstand something. This adds the flag `--quiet` that stores `False` to an internal variable called `verbose`. This lets us avoid negation in front of each verbose print we do. If we want to have `--verbose` for consistency we just add it as another argument via ```
parser.add_argument("--verbose", action="store_true", help="...")
```

Reply:
Sorry: I mean `--quiet` flag for the parser, but also quiet variable internally with negations where needed. 

And `--verbose` could be reserved for something really verbose (more output than by default without `--quiet`) in the future, but you can ignore this remark, the main nitpick was really about '--quiet` and mixing it with "verbose" internally (not a biggie really).

Reply:
But what's the point of that? The extra negations just add bulk and given that both `--verbose` and `--quiet` can be introduced there's no variable that is more correct. I chose it this way around to avoid the negation that just makes the logic harder to read.

Reply:
Ok, it's probably a matter of personal preference; my only point is that *if* being verbose/chatty is the default state of the tool, then I probably wouldn't create a flag that's positive by default. But again, it's a nitpick and a personal preference, not a blocker by any means, and I think it's not worth further discussion ;)